www/templates/default/html/manager/AddDatetime.tpl.php

```diff
@@ -130,74 +135,92 @@
         <label for="room">Room</label>
         <input type="text" id="room" name="room" value="<?php echo $datetime->room; ?>" />
 
+        <?php #Don't show the box when the current eventdatetime is a recurring instance or current evendatetime is a recurring and multi-day type.
+            if ($context->recurrence_id == NULL && !($datetime->recurringtype != 'none' && $event_duration_type == 'multi_day')) : ?>
+            <label for="single-day-event"><span class="required">*</span> Event duration type</label><br>
```

Comment:
This doesn't make sense as a label anymore, as it is labeling this group of radio buttons and not an individual radio button. Would using a `<fieldset>` and a `<lengend>` make more sense and be more semantic here?

Example:
```
<fieldset>
<legend>Event duration type</legend>
<label><input type="radio" ...> Single Day </label>
<label><input type="radio" ...> Multi Day </label>
</fieldset>
```